code/controllers/subsystem/job.dm

```diff
@@ -155,8 +155,8 @@ SUBSYSTEM_DEF(job)
 		return FALSE
 	job.current_positions = max(0, job.current_positions - 1)
 
-/datum/controller/subsystem/job/proc/FindOccupationCandidates(datum/job/job, level, flag)
-	JobDebug("Running FOC, Job: [job], Level: [level], Flag: [flag]")
```

Comment:
Nothing uses this flag, so I removed it. Special_role doesn't include job roles and never should.